SoftLayer/CLI/ticket/__init__.py

```diff
@@ -64,6 +64,8 @@ def get_ticket_results(mgr, ticket_id, update_count=1):
 
         # NOTE(kmcdonald): Windows new-line characters need to be stripped out
         wrapped_entry += click.wrap_text(update['entry'].replace('\r', ''))
+        if is_json:
```

Comment:
so `click.wrapp_text` is where the extra new lines are coming from. 

So lets do this instead of using regex:

```python
if is_json:
    wrapped_entry += updated['entry']
else:
    wrapped_entry += click.wrap_text(update['entry'].replace('\r', ''), width=120)
```